config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -40,6 +40,14 @@ jupyterhub:
     image:
       name: quay.io/arokem/nh-jhub-2022
       tag: "3d441bdb82f6"
+    nodeSelector:
+      2i2c.org/community: neurohackademy
+    cpu:
+      guarantee: 0.1
+      limit: 2
+    memory:
+      guarantee: 6G
+      limit: 8G
```

Comment:
Thank you for these suggestions, @consideRatio. I've implemented all of these, with some minor tweaks. I do think it's important to set non-trivial CPU guarantees - otherwise it only takes two users on the same node using upto their CPU limit to make sure all other users basically get just their CPU guarantee level of CPU. So i've set it to 0.5.

Reply:
Should we document that recommendation somewhere in our infra docs?